OpenRA.Mods.Common/UpdateRules/UpdatePath.cs

```diff
@@ -98,6 +98,7 @@ public class UpdatePath
 				new RemoveSequenceHasEmbeddedPalette(),
 				new RenameContrailWidth(),
 				new RemoveNegativeSequenceLength(),
+				new AddColorPickerValueRange(),
```

Comment:
All these rules are in the wrong directory - there should be a PR that renames 20121203 to 20230225 (see e.g. #20092, #18038).

Reply:
I included the renaming in this PR. I think it's fine as this is a must merge anyway